src/classes/tmDaoTrialFeatureRequest.cls

```diff
@@ -0,0 +1,120 @@
+        return accounts;
+    }
+
+    public static String getUserCountry(Id userId) {
+        List<User> u = [SELECT Id, ISO_Country__c FROM user WHERE Isactive = true AND Id = :userId];
+        if (!u.isEmpty() && !String.isEmpty(u[0].ISO_Country__c)) {
+            return u[0].ISO_Country__c;
+        } else {
+            return '';
+        }
+    }
+
+    public static Account updateAccount(Account acct) {
+        tmDaoTrialFeatureRequestHelper tfrHelper = new tmDaoTrialFeatureRequestHelper();
+        return tfrHelper.doUpdateAccountWithoutSharing(acct);
+    }
+
+    public static tmTrialFeatureRequest__c save(tmTrialFeatureRequest__c tfr, List<tmTrialFeatureRequestItem__c> tfrItems) {
+        tmDaoTrialFeatureRequestHelper tfrHelper = new tmDaoTrialFeatureRequestHelper();
+        return tfrHelper.doUpsertTrialFeatureRequestWithoutSharing(tfr, tfrItems);
+    }
+
+    public static void deleteItems(List<tmTrialFeatureRequestItem__c> tfrItems) {
+        tmDaoTrialFeatureRequestHelper tfrHelper = new tmDaoTrialFeatureRequestHelper();
+        tfrHelper.doDeleteTrialFeatureItems(tfrItems);
+    }
+
+
+    private without sharing class tmDaoTrialFeatureRequestHelper {
+        public tmTrialFeatureRequest__c doUpsertTrialFeatureRequestWithoutSharing(tmTrialFeatureRequest__c tfr, List<tmTrialFeatureRequestItem__c> tfrItems) {
+            tfr.AutoApprove__c = isUserProfileInAutoApprovalList();
+            upsert tfr;
+
+            if (tfrItems != null && tfrItems.size() > 0) {
+                delete [SELECT Id FROM tmTrialFeatureRequestItem__c WHERE TrialFeatureRequest__c = :tfr.Id];
+
+                for (tmTrialFeatureRequestItem__c tfrItem : tfrItems) {
+                    tfrItem.TrialFeatureRequest__c = tfr.Id;
+                }
+                insert tfrItems;
+            }
+            return tfr;
+        }
+
+        public void doDeleteTrialFeatureItems(List<tmTrialFeatureRequestItem__c> tfrItems) {
+            if (tfrItems != null && tfrItems.size() > 0) {
+                delete tfrItems;
+            }
+        }
+
+        public Account doUpdateAccountWithoutSharing(Account acct) {
+            Account a = new Account();
+            if (acct != null) {
+                a.Id = acct.Id;
+                a.BillingCountry = acct.BillingCountry;
+                if (String.isEmpty(acct.BillingStreet)) {
+                    a.BillingStreet = tmConstants.DEFAULT_ACCOUNT_BILLING_STREET;
+                }
+                if (String.isEmpty(acct.BillingCity)) {
+                    a.BillingCity = tmConstants.DEFAULT_ACCOUNT_BILLING_CITY;
+                }                
+                if (String.isEmpty(acct.BillingState)) {
+                    a.BillingState = tmConstants.DEFAULT_ACCOUNT_BILLING_STATE;
+                }
+                if (String.isEmpty(acct.BillingPostalCode)) {
+                    a.BillingPostalCode = tmConstants.DEFAULT_ACCOUNT_BILLING_ZIP;
+                }
+                update a;
```

Comment:
This DML statement is not surrounded by any try-catch block. Please add a try-catch block to ensure exceptions are handled properly.